manifests/setup/fluentbit-operator-clusterRole.yaml

```diff
@@ -89,12 +89,12 @@ rules:
   - apiGroups:
       - rbac.authorization.k8s.io
     resources:
-      - clusterrolebindings
+      - rolebindings
```

Comment:
we can also rename this file from `manifests/setup/fluentbit-operator-clusterRole.yaml` to `manifests/setup/fluentbit-operator-role.yaml`

Reply:
I think we still want to leave this file as `clusterRole`; I didn't modify the type, just changed some of the permissions underneath since the operator will create a more specific role binding per fluentbit daemonset instead of a cluster role.

A more advanced user could configure more granular permissions if they use the watch namespace flag though and create a `operator-role / binding` as you referenced.

Reply:
Agree